.changes/next-release/feature-AWSSDKforJavav2-8cde56c.json

```diff
@@ -0,0 +1,6 @@
+{
+    "category": "AWS SDK for Java v2", 
```

Comment:
nit: should it be S3 or maybe a separate entry for s3?

Reply:
Done